app/assets/javascripts/account/AccountController.js.coffee

```diff
@@ -157,15 +157,20 @@ AccountController = ($scope, $state, $document, $translate, AccountService, Shor
       if !_.isEmpty(data.application)
         # application for this listing was found
         if (data.application.status.match(/submitted/i))
-          $state.go('dahlia.my-applications', {skipConfirm: true, alreadySubmittedId: data.application.id})
+          doubleSubmit = !! ShortFormApplicationService.application.status.match(/submitted/i)
```

Comment:
agreed, was thinking about that. will take a look at refactoring before finishing this PR